### PR TITLE
Update logic around missing sample data in TDM [VS-997]

### PR DESCRIPTION
**Pull request by rsasch**
Also updated GATK docker to latest.

Latest run with missing columns [here](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/d9e5ec21-da22-42de-944c-a43ccf83e220) to show what "failing" looks like.
Quickstart integration failed because of https://broadworkbench.atlassian.net/browse/VS-1010.

### CODE REVIEW

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl [hunk 1]
```diff
@@ -293,7 +293,7 @@ task ExtractFilterTask {
   >>>
 
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_07_14"
+    docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_07_17"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateFilterSet.wdl:296]: Why did you need to update this docker? It looks like only the python (`variantstore`) one had code changes

**Review by gbggrant — APPROVED** [review]
LGTM

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:296]: to ensure that `bulk_ingest_staging`, the staging branch, is keeping up-to-date with java changes in `ah_var_store`

**Review by RoriCremer — APPROVED** [review]
